src/ModuleList/Article/Article.tsx

```diff
@@ -1,57 +1,35 @@
 import React from 'react';
-import { useParams } from 'react-router-dom';
+import { text } from './mockData';
+import ReactMarkdown from 'react-markdown';
+import { Heading } from '@chakra-ui/core';
+import { HeadingContainer, AuthorContainer, MarkdownWrapper, Div } from './ArticleStyles';
 
-// import { Carousel } from 'react-bootstrap';
 
-interface ExpectedParams {
-  moduleId?: string;
-  articleId?: string;
-}
+const data = {
+  'source': 'https://www.wired.co.uk/article/subsea-internet-cable-ship-boat',
+  'title': 'Ever wondered how underwater cables are laid? We take a trip on the ship that keeps us online',
+  'author': 'Matt Burgess',
+  'images': [
+      { 'file': '1.jpeg' },
+      { 'file': '2.jpeg' },
+      { 'file': '3.jpeg' },
+      { 'file': '4.jpeg' },
+      { 'file': '5.jpeg' },
+      { 'file': '6.jpeg' },
+      { 'file': '7.jpeg' },
+  ],
+};
+
 
 const Article = () => {
-  const { moduleId, articleId } = useParams<ExpectedParams>();
 
-  return (
-    <div>
-      <h1>Module: {moduleId}</h1>
-      <h2>Article: {articleId}</h2>
-      {/* <Carousel>
-        <Carousel.Item>
-          <img
-            className="d-block w-100"
-            src="https://images.idgesg.net/images/article/2017/08/networking-100733179-large.jpg"
-            alt="Third slide"
-          />
-          <Carousel.Caption>
-            <h3>First slide label</h3>
-            <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
-          </Carousel.Caption>
-        </Carousel.Item>
-        <Carousel.Item>
-          <img
-            className="d-block w-100"
-            src="https://images.idgesg.net/images/article/2017/08/networking-100733179-large.jpg"
-            alt="Third slide"
-          />
-          <Carousel.Caption>
-            <h3>Second slide label</h3>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-          </Carousel.Caption>
-        </Carousel.Item>
-        <Carousel.Item>
-          <img
-            className="d-block w-100"
-            src="https://images.idgesg.net/images/article/2017/08/networking-100733179-large.jpg"
-            alt="Third slide"
-          />
 
-          <Carousel.Caption>
-            <h3>Third slide label</h3>
-            <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
-          </Carousel.Caption>
-        </Carousel.Item>
-      </Carousel> */}
-    </div>
+  return (
+    <Div>
+      <HeadingContainer><Heading as='h1'size='2xl'>{data.title}</Heading></HeadingContainer>
+      <AuthorContainer><Heading as='h2'size='xl'>{data.author}</Heading></AuthorContainer>
```

Comment:
Per Tay's comment from yesterday's meeting, maybe add "Author: "?
```suggestion
      <HeadingContainer><Heading as='h1'size='2xl'>{data.title}</Heading></HeadingContainer>
      <AuthorContainer><Heading as='h2'size='xl'>{`Author: ${data.author}`}</Heading></AuthorContainer>
```